src/installer/corehost/cli/test/nativehost/host_context_test.h

```diff
@@ -57,6 +57,7 @@ namespace host_context_test
         const pal::char_t *config_path,
         int argc,
         const pal::char_t *argv[],
+        bool as_dotnet,
```

Comment:
Can we instead call this `launch_as_if_dotnet`? I assume that is the intent of this statement. Perhaps a comment would be sufficient. The statement of `as_dotnet` initially implied to me this isn't a .NET scenario rather than meaning the `dotnet` binary.

Reply:
Renamed to `launch_as_if_dotnet` and added a comment.